faq/databases-for-redis.mdx

```diff
@@ -0,0 +1,74 @@
+---
+meta:
+  title: Databases for Redis™
+  description: Discover Scaleway Managed Databases and how to access it for Redis™.
+content:
+  h1: Databases for Redis™
+  paragraph: Discover Scaleway Managed Databases and how to access it for Redis™.
+image: database-redis.webp
+---
+
+## What is Scaleway Databases for Redis™
+
+Databases for Redis™ is a low-latency caching solution. It allows you to easily set up a secure cache and lighten the load on your main database. Based on the in-memory data storage, Databases for Redis™ improves your application response time and helps you provide a better experience to your users. 
+The resource optimizes the speed of your requests as copies of the most frequently used data are stored in memory making them accessible in milliseconds. Learn more about Redis™ databases.
```

Comment:
I feel like "The resource" is not clear enough, maybe "Caching data" or "Using Database for Redis as a cache"